### PR TITLE
[All hosts] (dev program) Update info about the Office 365 subscription you can acquire via the Office 365 Developer Program. 

### PR DESCRIPTION
**Pull request by kbrandl**
Addresses internal work item # 3880698.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c9b7ad1](https://github.com/OfficeDev/office-js-docs-pr/commits/c9b7ad1cfcd5df50cae8b3a3bb827599523ad1c5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-vs-prerequisites.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/includes/quickstart-vs-prerequisites.md) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-add-ins-in-office-online.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/testing/debug-add-ins-in-office-online.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001151951110315-1545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DlgKa69cU4rtSfLijTh8xp9NdOn1m%2FgcIN70Xmxj5Dw%3D&st=2020-01-15T19%3A47%3A33Z&se=2020-02-15T19%3A52%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/create-sso-office-add-ins-aspnet.md [hunk 1]
```diff
@@ -19,7 +19,12 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 
 * [Office Developer Tools](https://www.visualstudio.com/features/office-tools-vs.aspx)
 
-* Office 365 (the subscription version of Office) account which you can get by joining the [Office 365 Developer Program](https://aka.ms/devprogramsignup) that includes a free 1 year subscription to Office 365. You should use the latest monthly version and build from the Insiders channel but you need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features, including SSO, is turned off for that build.
+* An Office 365 (the subscription version of Office) account. If you don't already have an Office 365 account, you can get a free, 90-day renewable Office 365 subscription by joining the [Office 365 Developer Program](https://aka.ms/devprogramsignup).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/create-sso-office-add-ins-aspnet.md]: Would it be useful to put this info into an include file?

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Because add-ins are developed using HTML and JavaScript, they are designed to wo
 
 If you have add-in that shows UI in a task pane or in a content add-in, you can debug an Office Add-in using Safari Web Inspector.
 
-To be able to debug Office Add-ins on Mac, you must have Mac OS High Sierra AND Mac Office Version: 16.9.1 (Build 18012504) or later. If you don't have an Office Mac build, you can get one by joining the [Office 365 Developer program](https://aka.ms/o365devprogram).
+To be able to debug Office Add-ins on Mac, you must have Mac OS High Sierra AND Mac Office Version: 16.9.1 (Build 18012504) or later. If you don't have an Office Mac build, you can get one by joining the [Office 365 Developer Program](https://aka.ms/o365devprogram).
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Is there a difference between https://aka.ms/o365devprogram and https://developer.microsoft.com/office/dev-program? Maybe we can use one or the other more consistently.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c79920e](https://github.com/OfficeDev/office-js-docs-pr/commits/c79920e561e9b89cedf3c2cd92fa450c18633625)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-vs-prerequisites.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/includes/quickstart-vs-prerequisites.md) | :white_check_mark:Succeeded | |
[docs/includes/sso-tutorial-prereqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/includes/sso-tutorial-prereqs.md) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/sso-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/quickstarts/sso-quickstart.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-add-ins-in-office-online.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/testing/debug-add-ins-in-office-online.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001152010575378-1545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=a%2BCKNdtq%2Bjwjh6u1JGsvJW1I9emz5zuROz8rvfFfdgc%3D&st=2020-01-15T20%3A07%3A06Z&se=2020-02-15T20%3A12%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0f3ec35](https://github.com/OfficeDev/office-js-docs-pr/commits/0f3ec355947087e6882cff05503feaaa223e0009)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-vs-prerequisites.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/includes/quickstart-vs-prerequisites.md) | :white_check_mark:Succeeded | |
[docs/includes/sso-tutorial-prereqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/includes/sso-tutorial-prereqs.md) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/sso-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/quickstarts/sso-quickstart.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-add-ins-in-office-online.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/testing/debug-add-ins-in-office-online.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-dev-program-info/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001152013247469-1545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7d2GrqFbfXbROnhWgTwzWd4bEXO9oYtc3qGfk0Fv5IE%3D&st=2020-01-15T20%3A09%3A32Z&se=2020-02-15T20%3A14%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT thanks for the feedback! I've moved the SSO-related prereqs (i.e., the two that are duplicated across three articles) into an include file and updated the Dev Program link to be consistent throughout.

**Pull request merged**